Source/Csla/Server/DataPortal.cs

```diff
@@ -577,17 +530,17 @@ internal void Complete(InterceptArgs e)
         Dashboard.CompleteCall(e);
       }
 
-      if (_interceptor != null)
-        _interceptor.Complete(e);
+      foreach (var interceptor in InterceptorList)
```

Comment:
In my implementation, I chose to iterate the interceptors backwards for the Complete method. This felt slightly more appropriate in that the earlier in the list they are, the more influence they are able to have on the result.